packages/manager/apps/telecom/src/app/telecom/telephony/billingAccount/guides/telecom-telephony-guides.html

```diff
@@ -3,51 +3,45 @@
         <oui-spinner></oui-spinner>
     </div>
 
-    <div class="row" data-ng-if="!$ctrl.loading.init">
-        <div class="col-md-6">
-            <oui-message data-type="info" data-ng-if="!$ctrl.hasGuides()">
-                <span data-translate="telephony_guides_empty"></span>
-            </oui-message>
+    <div data-ng-if="!$ctrl.loading.init">
+        <oui-message data-type="info" data-ng-if="!$ctrl.hasGuides()">
+            <span data-translate="telephony_guides_empty"></span>
+        </oui-message>
 
-            <section
-                class="widget-presentation"
-                data-ng-if="$ctrl.hasGuides()"
-                data-ng-repeat="section in $ctrl.guides.sections track by section.label"
-            >
-                <header class="widget-presentation-header">
-                    <h2
-                        class="widget-presentation-title"
-                        data-translate="{{:: section.label }}"
-                    ></h2>
-                </header>
-                <ul>
-                    <li
-                        data-ng-repeat="guide in section.guides track by guide.label"
+        <section
+            class="widget-presentation"
+            data-ng-if="$ctrl.hasGuides()"
+            data-ng-repeat="section in $ctrl.guides.sections track by section.label"
```

Comment:
maybe just track by index? i'am always afraid it breaks somehow if two guides have same label

Reply:
👍  I fixed one issue in prod related to this. You have added label in the constants file. If someone else introduces a duplicate label it will fail.